app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -215,7 +215,8 @@ def alert_build_edit_screen
 
     if params[:copy]  # If copying, create a new alert based on the original
       a = MiqAlert.find(params[:id])
-      @alert = MiqAlert.new(a.attributes)
+      # skip record id when copying attributes
+      @alert = MiqAlert.new(a.attributes.inject({}){|h,(k,v)| h[k]=v unless k == 'id'; h })
```

Comment:
you could use `Hash#reject` and even better, you can use `MiqAlert#close` which will skip the `id` for you.